trainscript.sh

```diff
@@ -13,14 +11,16 @@ counter=0;
 
```

Comment:
Can you change this to:
`# use relative url + ./gifZip/$counter.png`
As the file name is going to mess up the sort?